helm/kong-app/templates/_helpers.tpl

```diff
@@ -200,7 +200,7 @@ Create the ingress servicePort value string
 
 {{- define "kong.wait-for-db" -}}
 - name: wait-for-db
-  image: "{{ .Values.image.registry }}/{{ .Values.waitImage.repository }}:{{ .Values.waitImage.tag }}"
+  image: "{{ .Values.image.registry }}/{{ .Values.image.repository }}:{{ .Values.image.tag }}"
```

Comment:
```suggestion
  image: "{{ .Values.image.registry }}/{{ .Values.image.name }}:{{ .Values.image.tag }}"
```

This got added to `fmt` recently. I think name is clearer and shorter.

https://github.com/giantswarm/fmt/blob/master/helm/image_templating.md

Reply:
If you dont mind I will change all image format in following PR. There are several places to check and it is kind of urgent for now. Thanks for pointing me to the fmt, indeed I like more `name`

Reply:
@pipo02mix Yes sure the renaming isn't urgent.